panther_analysis_tool/main.py

```diff
@@ -700,6 +705,15 @@ def test_analysis(args: argparse.Namespace) -> Tuple[int, list]:
             f"No analysis in {args.path} matched filters {args.filter} - {args.filter_inverted}"
         ]
 
+    available_destinations = []
+    if args.available_destination:
+        available_destinations.extend(args.available_destination)
+
+    destinations_by_name = {
+        name: FakeDestination(destination_id=str(uuid4()), destination_display_name=name)
+        for name in available_destinations
+    }
+
```

Comment:
Ideally we shouldn't have to do this, but it's a nice workaround in the mean time

Reply:
Can you explain your concern? It's not only a workaround, it also facilitates more reliable testing of the `destinations` functions output, isn't that right? Maybe I am missing something, but you could also consider it an improvement 😄 .